### ISSUE TITLE
ACC-264.1 (Level A) _ Modify Modules _ Navigation menu _ Unnecessary info read for each 

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3149]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	Navigation menu
Affected Check Point:	ACC-264.1
Short text:	ACC-264.1 (Level A) _ Modify Modules _ Navigation menu _ Unnecessary info read for each 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Open JAWS 2024 screen reader and turn on VPC mode.
•	Now navigate to all the menu items from the left side navigation using arrow keys one at a time.

3. Issue Type: Manual
Observed Behavior:
On accessing all the menu items from the left side navigation using arrow keys one at a time , information ‘Selected’ is  announced by the screen reader even if the particular list item is not selected by the user.
Screenshot:
 
Case 2: 
 

Expected Behavior:
On accessing all the menu items from the left side navigation using arrow keys one at a time , information ‘Selected’ must be announced by the screen reader only for the list item which is currently selected along with other necessary information.

Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-264.1 (Level A) _ Modify Modules _ Navigation menu _ Unnecessary info read for each.docx](https://github.com/user-attachments/files/16660118/ACC-264.1.Level.A._.Modify.Modules._.Navigation.menu._.Unnecessary.info.read.for.each.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16660119/Issue.processing-09.2021.002.docx)

**Comment by chriskari** [comment]
looks like it was resolved by the UI5v2 migration
